services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -264,8 +264,30 @@ async def create_project(  # pylint: disable=too-many-arguments,too-many-branche
     project_nodes = None
     try:
         task_progress.update(message="creating new study...")
+
+        workspace_id = None
+        folder_id = None
+        if predefined_project:
+            if workspace_id := predefined_project.get("workspaceId", None):
+                await check_user_workspace_access(
+                    request.app,
+                    user_id=user_id,
+                    workspace_id=workspace_id,
+                    product_name=product_name,
+                    permission="write",
+                )
+            if folder_id := predefined_project.get("folderId", None):
+                # Check user has access to folder
+                await folders_db.get_for_user_or_workspace(
```

Comment:
her same. instead of having this comment. why not have the function that does it?